### PR TITLE
Feat/#394 access key hookup

### PR DESCRIPTION
**Pull request by ciaranschutte**
**Description of changes**

Hooks up access key ui to data

**Type of Change**

- [ ] Bug
- [ ] Styling
- [x] New Feature

**Checklist before requesting review:**

- [x] Matches design:

  - component sizes, spacing, and styles
  - font size, weight, colour
  - spelling has been double checked

- [ ] New uikit components have Storybook stories
- [ ] Feature is minimally responsive.
- [x] Manual testing of UI feature.
- [ ] Selenium test is completed and passing.

### CODE REVIEW

**Comment by ciaranschutte** [comment]
Hopefully you guys have a suggestion to clean up the hooks.

Problem is we either display the key we have retrieved
Or we make a request for a new one and show that

I can't figure out how to put `useQuery` hook returned data into state so we only have one key in state at a time

**Comment by d8660091** [comment]
I think it's better to use https://www.npmjs.com/package/@apollo/react-hooks for the newly added code.
And we can use onCompleted option to set the returned data to the same state

**Comment by ciaranschutte** [comment]
> I think it's better to use https://www.npmjs.com/package/@apollo/react-hooks for the newly added code.
Will need to do a full migration to this package from `react-apollo-hooks`

**Comment by ciaranschutte** [comment]
@joneubank @d8660091 ok to merge?

File: components/pages/user/AccessKeyBox/index.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+import ClipboardCopyField from 'uikit/ClipboardCopyField';
+import EGO_ACCESS_KEY from './EGO_ACCESS_KEY.gql';
+import GENERATE_EGO_ACCESS_KEY from './GENERATE_EGO_ACCESS_KEY.gql';
+import { useQuery, useMutation } from 'react-apollo-hooks';
+
+export default function AccessTokenBox() {
+  const { data: { accessKey } = {}, loading } = useQuery(EGO_ACCESS_KEY);
+
+  const [generatedKey, setGeneratedKey] = React.useState(null);
+  const [isGeneratingKey, setIsGeneratingKey] = React.useState(false);
```
Comment (ciaranschutte) [inline components/pages/user/AccessKeyBox/index.js]: @hlminh2000 shouldn't have two keys in state. makes all the logic very confusing.

**Pull request merged**